test-partner/partner.yaml

```diff
@@ -2,18 +2,18 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: partner
-  namespace: tnf
+  name: tnfpartner
+  namespace: default
```

Comment:
use {{ DEFAULT_NAMESPACE }} here as well (and other similar places) for replacement with mustache

Reply:
what's the reason for this ? it should always be default.
Also, the test-partner is full example, not template. the template is in local-test-infra folder.

Reply:
I'll do it anyway.. it would be useful in case we want to switch the "default" namespace in the future.

Reply:
fixed